src/stylesheets/elements/_buttons.scss

```diff
@@ -44,8 +44,8 @@ $button-stroke: inset 0 0 0 units($theme-button-stroke-width);
     color: color("white");
   }
 
-  &:focus,
-  &.usa-focus {
+  &:not([disabled]):focus,
+  &:not([disabled]).usa-focus {
```

Comment:
why repeat when you can... 

```
&:not([disabled]) {
  &:focus,
  &.usa-focus {
   ...
  }
}
```
?

Reply:
Stylistic choice, seemed more readable to me.

Reply:
Yeah, makes sense. We can fold it into the next block of work.